pkg/chunked/storage_linux.go

```diff
@@ -58,6 +58,20 @@ const (
 	copyGoRoutines = 32
 )
 
+// fileMetadata is a wrapper around internal.FileMetadata with additional private fields that
+// are not part of the TOC document.
+type fileMetadata struct {
+	internal.FileMetadata
+
+	// Chunks are computed by mergeTOCEntries.
+	Chunks []*internal.FileMetadata `json:"-"`
```

Comment:
- The internal fields can be package-private
- I read the the `json` annotation to suggest that this struct can be marshaled/unmarshaled; that’s AFAICS not the case, so dropping the annotation would make this easier to follow.